### PR TITLE
Add SVG icons and scaling factor

### PR DESCRIPTION
**Pull request by roscale**
- Scaling goes from 0.01 to 13.7. 2 decimals for scaling less than 1, and 1 decimal for scaling greater than 1.
- Added SVG icons alongside the already existing PNG ones
- Added the GYW icon

### CODE REVIEW

**Comment by roscale** [comment]
If you wonder why the width and height are now passed in the constructor, it's because the GYW icon isn't square like the rest of them.

**Review by TheToto42 — APPROVED** [review]
Like for pygyw, add a word to explain the scale factor, otherwise it's good

File: README.md [hunk 1]
```diff
@@ -139,13 +139,14 @@ Note that through this drawing, you can change the background color.
 
 ### How can I have bigger icons?
 
-For now, only a few 48x48 icons are supported. New icons and different sizes will be added in future versions.
+IconDrawing has a scale parameter that allows you to scale the icon up or down. If the icon is 48x48, a scale factor of 2.0 will result in a 96x96 icon. The scale factor has a range between 0.01 and 13.7. Any scale factor outside of this range will be clamped to the nearest value within the range.
+
+```dart
```
Comment (TheToto42) [inline README.md:144]: What is this leftover ? Shouldn't be removed ?
```suggestion
```

Comment (roscale) [reply README.md:144]: Whoops. Yes it should.

**Pull request merged**